api-server/app.js

```diff
@@ -0,0 +1,24 @@
+const express = require("express");
+const cors = require("cors");
+const helmet = require("helmet");
+const path = require("path");
+
+const app = express();
+
+const db = require("./db");
+
+const cats = require("./routes/cats");
+const sharks = require("./routes/sharks");
+
+app.use(express.static(path.resolve(__dirname, '../client/build')));
+app.use(cors());
+app.use(helmet());
+
+app.use("/api", cats(db));
+app.use("/api", sharks(db));
```

Comment:
What is another way you could design these endpoints to make it "more RESTful"?

Reply:
There are definitely a few different ways that I could have set up my endpoints for this project. While working on the project I had made an assumption that due to the limited requirements as far as what I needed to pull, I did not prioritize complexity on the back end, however, I totally understand that there were limitations with the structure of the simple cats and sharks endpoints that I used. 

I think a simple approach that would make the endpoints more RESTful would to instead view the fundamental resource that is being shared as pictures or posts instead of grouping them by animal category. Under this model, REST API endpoints could then be "API/pics/picId:". Using this model, I could then have accessed the different groups of pictures by adding query parameters to get requests. An example of this could be "API/pics?type=cats&limit=10". By adding query parameters of type along with adding an ability to filter and paginate queries, this would more closely replicate a database that had a larger volume.